RULES.md

```diff
@@ -390,6 +391,12 @@ A location that must have `parent_station` field does not have it. The following
 
 Start and end times have been found to be out-of-order in GTFS file `frequencies.txt`.
 
+<a name="StartAndEndTimeEqualNotice"/>
+
+### StartAndEndTimeEqualNotice
+
+Start and end times have been found to be equal in GTFS file `frequencies.txt`.
```

Comment:
It would be good to add something here about why this is a warning. Maybe:

>The GTFS spec is currently unclear how this case should be handled (e.g., is it a trip that circulates once?). It is recommended to use a trip not defined via `frequencies.txt` for this case.

Reply:
Thanks! Will do.